packages/react/src/drafts/IssueLabel/IssueLabel.tsx

```diff
@@ -29,7 +31,7 @@ export interface IssueLabelProps extends React.PropsWithChildren {
   /**
    *
    */
-  fillColor?: string
+  fillColor?: `#{string}`
```

Comment:
I think this will help with some of the errors coming into `IssueLabel` from TS since it's missing the `$` for interpolation

```suggestion
  fillColor?: `#${string}`
```